ipaclient/plugins/migration.py

```diff
@@ -77,3 +77,4 @@ def output_for_cli(self, textui, result, ldapuri, **options):
                                ldapuri)
             return 1
         textui.print_plain(unicode(self.pwd_migration_msg))
+        return None
```

Comment:
`cli` class expects that `output_for_cli` returns either error code or something that evaluates into a `False`. In the latter case we get the result silenced (`cli.run()` returns `0`). Probably better to change `return None` to `return 0` for consistency.